1-js/06-advanced-functions/02-rest-parameters-spread-operator/article.md

```diff
@@ -190,56 +189,55 @@ let arr2 = [8, 9, 15];
 
-For this particular task we could also use `Array.from`, because it converts an iterable (like a string) into an array:
+Para essa tarefa em particular nós também poderíamos usar `Array.from`, porque ele converte um iterável (como uma string) em um _array_.
 
 ```js run
-let str = "Hello";
+let str = "Olá";
 
-// Array.from converts an iterable into an array
-alert( Array.from(str) ); // H,e,l,l,o
+// Array.from converte um interável em um _array_
+alert( Array.from(str) ); // O,l,á
 ```
 
-The result is the same as `[...str]`.
-
-But there's a subtle difference between `Array.from(obj)` and `[...obj]`:
+O resultado é o mesmo que `...[str]`.
 
-- `Array.from` operates on both array-likes and iterables.
-- The spread operator operates only on iterables.
+Mas há uma diferença sútil entre `Array.from(obj)` e `[...obj]`:
 
-So, for the task of turning something into an array, `Array.from` tends to be more universal.
+- `Array.from` opera tanto em listas-genéricas _(array-likes)_ como em iteráveis.
+- O operador spread opera somente em iteráveis.
 
+Então, para a tarefa de transformar algo em um _array_, `Array.from` tende a ser uma solução mais universal.
 
-## Summary
+## Sumário
 
-When we see `"..."` in the code, it is either rest parameters or the spread operator.
+Quando nos depararmos com `"..."` no código, estamos falando de parâmetros _rest_ ou operador _spread_.
 
-There's an easy way to distinguish between them:
+Existe uma forma fácil para distinguir entre eles:
 
-- When `...` is at the end of function parameters, it's "rest parameters" and gathers the rest of the list of arguments into an array.
-- When `...` occurs in a function call or alike, it's called a "spread operator" and expands an array into a list.
+- Quando `...` está no final dos parâmetros da função, é "parâmetros _rest_" e reune o restante da lista de argumentos em um _array_.
+- Quando `...` ocorre em uma chamada de função ou similar, é chamado de "operador _spread_" e expande um _array_ em uma lista.
 
-Use patterns:
+Padrões de uso:
 
-- Rest parameters are used to create functions that accept any number of arguments.
-- The spread operator is used to pass an array to functions that normally require a list of many arguments.
+- Operadores _rest_ são usados para criar funções que aceitem qualquer número de argumentos.
+- O operador _spread_ é usado para passar um _array_ em funções que normalmente requerem uma lista de muitos argumentos.
 
-Together they help to travel between a list and an array of parameters with ease.
+Juntos, eles nos ajudam a efetuar a transição entre uma lista e um _array_ de parâmetros com facilidade.
 
-All arguments of a function call are also available in "old-style" `arguments`: array-like iterable object.
+Todos os argumentos de uma chamada de função também estão disponíveis na "moda antiga" `arguments`: objeto iterável tipo lista-genérica _(array-like)_.
```

Comment:
Please,

- '... objeto iterável do tipo lista-genérica (_array-like_).'
- Would you also move the emphasis  to inside the parentheses?
Thanks.